docs/developer-guide/map-query-parameters.md

```diff
@@ -371,3 +371,29 @@ Data of resulting layer can be additionally filtered by passing "CQL_FILTER" int
 GET `#/viewer/config?actions=[{"type":"CATALOG:ADD_LAYERS_FROM_CATALOGS","layers":["layer1","layer2","workspace:externallayername"],"sources":["catalog1","catalog2",{"type":"WMS","url":"https://example.com/wms"}],"options": [{"params":{"CQL_FILTER":"NAME='value'"}}, {}, {"params":{"CQL_FILTER":"NAME='value2'"}}]}]`
 
 Number of objects passed to the options can be different to the number of layers, in this case options will be applied to the first X layers, where X is the length of options array.
+
+The 3D tiles service endpoint does not contain a default property for the name of the layer and it returns only a single layer. Below some ways to correctly request the layer contained inside 3D tiles service:
+
+- Use the `url` property to get the layer
+
+```json
+{
+    "type": "CATALOG:ADD_LAYERS_FROM_CATALOGS",
+    "layers": ["https://example.com/tileset-pathname/tileset.json"],
+    "sources": [{ "type":"3dtiles", "url":"https://example.com/tileset-pathname/tileset.json" }]
+}
+```
```

Comment:
I tried this and I had a "aa5" title, but I don't know where it comes from, can you explain?

Reply:
New proposed configuration uses the layer entry as title of the new layer, no more multiple approaches

> The 3D tiles service endpoint does not contain a default property for the name of the layer and it returns only a single record for this reason the name used in the layers array will be used to apply the title to the added 3D Tiles layer:


```json
{
    "type": "CATALOG:ADD_LAYERS_FROM_CATALOGS",
    "layers": ["My 3D Tiles Layer"],
    "sources": [{ "type":"3dtiles", "url":"https://example.com/tileset-pathname/tileset.json" }]
}
```

GET: `#/viewer/config?actions=[{"type":"CATALOG:ADD_LAYERS_FROM_CATALOGS","layers":["My 3D Tiles Layer"],"sources":[{"type":"3dtiles","url":"https://example.com/tileset-pathname/tileset.json"}]}]`

Reply:
can layers elements be objects? I should follow the same system of other MapStore. Or maybe options can contain more settings (e.g. localized title?).

I should be as much as possible consistent with the rest of the layers.